lnrpc/lightning.proto

```diff
@@ -2390,6 +2387,8 @@ message PendingChannelsResponse {
         transaction. This value can later be updated once the channel is open.
         */
         int64 fee_per_kw = 6;
+
+        reserved 2;
```

Comment:
Can you add a comment to mention what this previously was and to not re-use this argument number please?

Reply:
+1